spec/migrations/20181001131632_add_conversion_host_id_to_miq_request_tasks_spec.rb

```diff
@@ -0,0 +1,56 @@
+require_migration
+
+describe AddConversionHostIdToMiqRequestTasks do
+  let(:task_stub) { migration_stub(:MiqRequestTask) }
+  let(:host_stub) { migration_stub(:Host) }
+  let(:conversion_host_stub) { migration_stub(:ConversionHost) }
+  let(:host_id) { 42 }
```

Comment:
This looks weird, 42 is not going to be in the right region or worse if the region is 0 it might be a valid host?
How about getting the host_id from a host_stub then destroying the record.

Reply:
Should be good.